### PR TITLE
Update configure.md

### PR DESCRIPTION
**Pull request by edwilliams16**
Clarified the meanings of DEFAULT_MAX_ACCELERATION, DEFAULT_ACCELERATION, DEFAULT_XYJERK etc.

### CODE REVIEW

File: _articles/getting-started/configure.md [hunk 1]
```diff
@@ -639,7 +639,7 @@ Pulled from the above setting, on `M204` command.
 #define DEFAULT_XYJERK                15.0    // (mm/sec)
 {% endhighlight %}
 
-Jerk works in conjunction with acceleration above. Both of acceleration and jerk will affect your print quality too especially cube and round shape.
+Jerk works in conjunction with acceleration above. Jerk is the maximum change in velocity (in mm/sec) that can be commanded to occur instantaneously. Both acceleration and jerk can affect your print quality. Set too low, the extruder will linger too long at points of direction change, causing for instance, blobbing at the corners of a cube. Setting too high demands too much of your printers mechanics and you may see ringing artifacts or dropped steps.
```
Comment (thinkyhead) [inline _articles/getting-started/configure.md]: Suggested adjustment…
```
Jerk works in conjunction with acceleration. Jerk is the maximum change in velocity (in mm/sec) that can occur instantaneously. Both acceleration and jerk affect print quality. Set too low, the extruder will linger too long at points of direction change, causing, for instance, blobs at the corners of a cube. Set too high, direction changes put too much torque on a printer's mechanics, and you may see ringing artifacts or dropped steps.
```

Comment (edwilliams16) [inline _articles/getting-started/configure.md]: Looks improved to me.

Comment (edwilliams16) [inline _articles/getting-started/configure.md]: I merged in thinkyhead's suggestion and updated the boards list.

**Comment by jbrazio** [comment]
@edwilliams16 you should add your nickname to the front matter of the pages you edit as a `contrib`, take as an example the front matter for `configure.md`

```
---
layout: articles
author: Sarf2k4
contrib: paulusjacobus, jbrazio

title:        'How to configure'
description:  'This howto will guide the user on how to configure Marlin for their needs'
category:     [ getting-started, needs-review ]

toc:
  selectors:  h1,h2,h3,h4
---
```

**Comment by edwilliams16** [comment]
Will do.  Wasn't clear if that was my prerogative.

**Pull request merged**